app/tests/tests_10/test_collection_assets_endpoint.py

```diff
@@ -316,29 +316,15 @@ def test_asset_endpoint_patch_put_href(self):
         asset_sample = self.asset.copy()
 
         put_payload = asset_sample.get_json('put')
-        put_payload['href'] = 'https://testserver/non-existing-asset'
-        patch_payload = {'href': 'https://testserver/non-existing-asset'}
+        put_payload['href'] = 'https://testserver/external-asset'
```

Comment:
Test doesn't seems to be fully complete in this class we should have

-`test_asset_endpoint_patch_put_href_to_external_href`. Actual test case like you modified it, meaning the current asset is internal and is modified to external. Making also sure that before the update you have an internal href.
- `test_asset_endpoint_patch_put_external_href` similar as above the the asset should already be an external asset where you modify the href, so making sure that before the put/patch you have href A and after you have href B.